src/components/_inputs/js/InputBoolean.js

```diff
@@ -0,0 +1,25 @@
+export default {
+  name: 'InputText',
+  props: {
+    schema: {
+      type: Object
+    },
+    name: {
+      type: String,
+      required: true
+    },
+    value: {
+      type: Boolean
+    }
+  },
+  computed: {
+    localValue: {
+      get () {
+        return this.value
```

Comment:
Não sei se você já está tratando o `default` do schema em outro lugar, está?

Se não estiver:

```suggestion
        return typeof this.value === 'boolean' ? this.value : Boolean(this.schema.default)
```

O que acha?